options/linux/include/asm/ioctls.h

```diff
@@ -6,8 +6,9 @@
 #define TCGETS 0x5401
 #define TCSETS 0x5402
 #define TIOCSCTTY 0x540E
-#define TIOCNOTTY 0x5422
+#define TIOCSTI 0x5412
```

Comment:
Was this one wrong before?

Reply:
No, I moved the `TIOCNOTTY` define to the bottom of the list, as it has the highest number. The value is still the same, the only change is that the list is now sorted.

Reply:
Ah okay I just can't read I see hahahahah